google/ads/google_ads/v2/__init__.py

```diff
@@ -265,6 +263,14 @@ def __getattr__(name):  # Requires Python >= 3.7
                      {'__doc__': sub_mod_class.__doc__, 'enums': enums})
         globals()[name] = klass
         return klass
+    elif name == 'enums':
+        module = importlib.import_module('google.ads.google_ads.v3.services.enums')
```

Comment:
This is the v2 init so why are you importing the v3 enums?

Reply:
Ah just a typo, fixed